src/platform/packages/shared/kbn-scout/src/playwright/fixtures/worker/core_fixtures.ts

```diff
@@ -56,16 +57,25 @@ export const coreWorkerFixtures = base.extend<
     esClient: Client;
     kbnClient: KbnClient;
     samlAuth: SamlAuth;
+    flags: Flags;
   }
 >({
   // Provides a scoped logger instance for each worker to use in fixtures and tests.
   // For parallel workers logger context is matching worker index+1, e.g. '[scout-worker-1]', '[scout-worker-2]', etc.
+  flags: [
+    ({}, use) => {
+      const flags = getFlags(process.argv.slice(2));
```

Comment:
I think we can't do it this way: we run Scout with `node scripts/scout` cli and that one defines logger level on global level, but it is not passed into Playwright command. Playwright `--debug` flag starts inspector, so we don't want to pass the same flag from entry.

And I'm not sure about adding extra variable in PW command here https://github.com/elastic/kibana/blob/9200fd8dcf79d02f544aeb5b91505a25a111a120/src/platform/packages/shared/kbn-scout/src/playwright/runner/run_tests.ts#L138-L144 Maybe you have other ideas how to unify it?

Reply:
yeah but I also want to do that when running locally